makefile

```diff
@@ -0,0 +1,24 @@
+# By default, this makefile will run everything inside a Docker container.
+# If you're a developer and want to run everything with your local `cargo`:
+# - Creates a `.env` file in the same folder as this `makefile`
+# - Add a line in `.env` with `LOCAL_DEV=true`
+-include .env
+
+ifeq ($(LOCAL_DEV),true)
+	SH=sh -c
```

Comment:
Here is a proposal to use the local shell : #6 

And I can't help but think that we should use the docker command only on CI (and skip the use as dev without a rust installed).
Same for the install of rustfmt.

Maybe script-helpers could be in makefile (`setup` and maybe `docker_setup` targets), but it's OK to expect the platform to be ready before running make targets IMO.